hips/draw/simple.py

```diff
@@ -116,6 +117,32 @@ def fetch_tiles(healpix_pixel_indices: np.ndarray, order: int, hips_survey: Hips
         yield tile
 
 
+def get_hips_order(geometry: WCSGeometry, hips_survey: HipsSurveyProperties) -> int:
+    """Compute the tile order suited for the given geometry and hips_survey"""
```

Comment:
Add `Parameters` and `Returns` sections documenting the inputs and outputs.
Maybe even an `Examples` section with one working example so that users can see how it works in the docs?